obal/data/modules/koji_build.py

```diff
@@ -0,0 +1,69 @@
+"""
+Release a package to koji
+"""
+
+import re
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.koji_wrapper import koji, KojiCommandError # pylint:disable=import-error,no-name-in-module
+from ansible.module_utils.obal import get_whitelist_status  # pylint:disable=import-error,no-name-in-module
+
+
+def main():
+    """
+    Build a package in koji
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            srpm=dict(type='path', required=True),
+            scratch=dict(type='bool', required=False, default=False),
+            tag=dict(type='str', required=True),
+            nevr=dict(type='str', required=True),
+            package=dict(type='str', required=True),
+            tag_check=dict(type='bool', required=False, default=True),
+            koji_executable=dict(type='str', required=False)
+        )
+    )
+
+    tag = module.params['tag']
+    scratch = module.params['scratch']
+    srpm = module.params['srpm']
+    nevr = module.params['nevr']
+    package = module.params['package']
+    koji_executable = module.params['koji_executable']
+
+    if module.params['tag_check']:
+        if not get_whitelist_status(module.params['koji_executable'], tag, package):
```

Comment:
`module.params['koji_executable']` can be `None`, and I don't see any assertions for `koji list-pkgs` in the log below?

Reply:
(also, `get_whitelist_status` doesn't use the `koji` wrapper we have in `obal/data/module_utils/koji_wrapper.py`, which would have saved you from this issue)